net-im/slack-bin/slack-bin-2.1.2.ebuild

```diff
@@ -1,63 +1,61 @@
-# Copyright 1999-2016 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
 
-inherit unpacker gnome2-utils
+MY_PN="${PN/-bin/}"
+BASE_URI="https://downloads.slack-edge.com/linux_releases/${MY_PN}-desktop-${PV}-_arch_.deb"
 
-QA_PREBUILT="
-	opt/slack/slack
-	opt/slack/resources/app.asar.unpacked/node_modules/*
-	opt/slack/libnode.so
-	opt/slack/libgcrypt.so.11
-	opt/slack/libffmpeg.so
-	opt/slack/libCallsCore.so
-"
+inherit eutils gnome2-utils unpacker
 
 DESCRIPTION="Team collaboration tool"
 HOMEPAGE="http://www.slack.com/"
+SRC_URI="x86? ( ${BASE_URI/_arch_/i386} )
+	amd64? ( ${BASE_URI/_arch_/amd64} )"
 
-MY_PN="${PN/-bin/}"
-BASE_URI="https://downloads.slack-edge.com/linux_releases/${MY_PN}-desktop-${PV}-_arch_.deb"
-
-SRC_URI="
-	x86? ( ${BASE_URI/_arch_/i386} )
-	amd64? ( ${BASE_URI/_arch_/amd64} )
-"
-
-LICENSE="MIT Apache-2.0 BSD ISC LGPL-2 AFL-2.1 public-domain WTFPL-2 Artistic-2 no-source-code"
+LICENSE="all-rights-reserved"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
 IUSE=""
-RESTRICT="mirror"
+RESTRICT="bindist mirror"
 
-RDEPEND="x11-libs/gtk+:2
-	x11-libs/libnotify
-	x11-libs/libXtst
-	x11-libs/pango
-	x11-libs/cairo[xcb]
-	media-libs/alsa-lib
-	media-libs/harfbuzz[graphite]
-	media-libs/libcanberra[gtk]
-	dev-libs/nss
+RDEPEND="dev-libs/atk:0
+	dev-libs/expat:0
 	dev-libs/glib:2
-	dev-libs/atk
-	gnome-base/libgnome-keyring
+	dev-libs/nspr:0
+	dev-libs/nss:0
 	gnome-base/gconf:2
-	sys-apps/dbus
-	net-print/cups[ssl]
-	net-misc/curl
-	virtual/udev
-	virtual/libc
-	virtual/libffi
-"
-DEPEND="${RDEPEND}"
+	gnome-base/libgnome-keyring:0
+	media-libs/alsa-lib:0
+	media-libs/fontconfig:1.0
+	media-libs/freetype:2
+	net-misc/curl:0
+	net-print/cups:0
+	sys-apps/dbus:0
+	x11-libs/cairo:0
+	x11-libs/gdk-pixbuf:2
+	x11-libs/gtk+:2
+	x11-libs/libX11:0
+	x11-libs/libXcomposite:0
+	x11-libs/libXcursor:0
+	x11-libs/libXdamage:0
+	x11-libs/libXext:0
+	x11-libs/libXfixes:0
+	x11-libs/libXi:0
+	x11-libs/libXrandr:0
+	x11-libs/libXrender:0
+	x11-libs/libXScrnSaver:0
+	x11-libs/libXtst:0
+	x11-libs/pango:0"
 
-S="${WORKDIR}"
+QA_PREBUILT="opt/slack/slack
+	opt/slack/resources/app.asar.unpacked/node_modules/*
+	opt/slack/libnode.so
+	opt/slack/libgcrypt.so.11
```

Comment:
Any reason not to unbundle it? We have `dev-libs/libgcrypt:11` for this library specifically.